.github/actions/run-interop-ping-test/action.yml

```diff
@@ -0,0 +1,78 @@
+name: 'libp2p ping interop test'
+description: 'Run the libp2p ping interoperability test suite'
+inputs:
+  test-filter:
+    description: "Filter which tests to run out of the created matrix"
+    required: false
+    default: ""
+  extra-versions:
+    description: "Space-separated paths to JSON files describing additional images"
+    required: false
+    default: ""
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        WORK_DIR=$(realpath "${{ github.action_path }}/../../../multidim-interop")
+        echo "WORK_DIR=$WORK_DIR" >> $GITHUB_OUTPUT
+      shell: bash
+      id: find-workdir
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 17
+
+    - name: Expose GitHub Runtime # Needed for docker buildx to cache properly (See https://docs.docker.com/build/cache/backends/gha/#authentication)
+      uses: crazy-max/ghaction-github-runtime@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Install deps
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: npm ci
+      shell: bash
+
+    - name: Build images
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: make
+      shell: bash
+
+    - name: Run the test
+#      timeout-minutes: 30
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: WORKER_COUNT=2 npm run test -- --extra-version=${{ inputs.extra-versions }} --name-filter=${{ inputs.test-filter }}
+      shell: bash
+
+    - name: Print the results
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: cat results.csv
+      shell: bash
```

Comment:
We don't need to specify bash everywhere, do we? This is default for linux/mac. I guess we'd need to specify this if this is run in windows (but I bet other things would break).

Can this be specified once at the top? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrun

Reply:
Their example sets shell every time as well: https://docs.github.com/en/actions/creating-actions/creating-a-composite-action

Reply:
This is required: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsshell.

Nvm